torch/_dynamo/variables/base.py

```diff
@@ -326,6 +326,13 @@ def call_method(
             return self.var_getattr(tx, args[0].as_python_constant()).add_options(
                 self, args[0]
             )
+        elif (
+            name == "__contains__"
+            and hasattr(self.value, "__len__")
+            and len(self.value) == 0
+        ):
+            return variables.ConstantVariable(False, **VariableTracker.propagate(self))
```

Comment:
This is sus.

```
class A:
    def __contains__(self, x):
        return True

print(2 in A())
```

prints True. len protocol has nothing to do with this.

Reply:
Yeah there was hack I did that's no longer needed with the handling of '__get__'